source/_docs/bots-and-indexing.md

```diff
@@ -50,7 +50,7 @@ Additionally, Pantheon's edge layer adds the [`X-Robots-Tag: noindex` HTTP heade
 
 The `pantheonsite.io` domains are intended for development use and cannot be used for production. Robots.txt is only visible on Live with a custom domain, and is not available on Dev or Test. Adding sub-domains (i.e. `dev.example.com`, `test.example.com`) for DEV or TEST  will remove the `X-Robots-Tag: noindex` header only, but still serve the custom robots.txt from the platform domain.
 
-[Site Auditor](https://raven.zendesk.com/hc/en-us/articles/202346870){.external} (aka RavenCrawler) can access platform domains like `test-example.pantheonsite.io` to support pre-release SEO testing. If you’re testing links or SEO with other tools, you may request the addition of the tool to `robots.txt` by contacting support. Otherwise, you can assign a test or beta domain to the Live environment and test your links following the alternative domain. In addition, if you run SEO toolsets locally, you can utilize an `/etc/hosts` file entry on your local development box to spoof your production domain on Pantheon.
+[Site Auditor](https://raven.zendesk.com/hc/en-us/articles/202346870){.external} (aka RavenCrawler) can access platform domains like `test-example.pantheonsite.io` to support pre-release SEO testing. If you’re testing links or SEO with other tools, you may request the addition of the tool to `robots.txt` by [contacting support](/docs/support/#feature-requests) to create a feature request. Otherwise, you can assign a test or beta domain to the Live environment and test your links following the alternative domain. In addition, if you run SEO toolsets locally, you can utilize an `/etc/hosts` file entry on your local development box to spoof your production domain on Pantheon.
```

Comment:
What's a test or beta domain? 

> Otherwise, you can assign a test or beta domain to the Live environment and test your links following the alternative domain.

Suggest using less jargon and include an example, like: 

> Otherwise, you can connect a custom domain to the Live environment for the purpose of testing links in production (e.g., `seo.example.com`).

Reply:
It would be nice to provide a link to instructions on how to modify local hosts file, but the places we provide that are not easily linked. 

 [Test Locally Before Updating DNS (optional) - dropdown in the launch guide](https://pantheon.io/docs/guides/launch/domains/)

[Upgrade your site - in global cdn doc](https://pantheon.io/docs/global-cdn/#upgrade-your-site)

Reply:
Should the domain be removed after tests? If they leave it in place, are there any implications (e.g., duplicate content dings for seo?)

Reply:
like your example....test or beta could be "non-production domain"

Reply:
+1 to make Test locally using hosts file easy to link to. I'm drafting another doc where I'd like to link to it as well. Can someone please create a follow-up issue if it's not done as part of this work?

Reply:
https://github.com/pantheon-systems/documentation/issues/3778